src/store/indexeddb-backend.ts

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2021 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { ISavedSync } from "./index";
+import { IEvent, IStartClientOpts, ISyncResponse } from "..";
+
+export interface IIndexeddbBackend {
```

Comment:
The types in the next file use the spelling `IDB` as opposed to `Idb`, so I feel like this would match more as...

```suggestion
export interface IIndexedDBBackend {
```

...but feel free to ignore if you feel strongly. Even though we use write e.g. `Url`, we seem to also write `IndexedDB` as the spelling in most places.

Reply:
oh yeah, not sure how I messed that one up